src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -13,19 +13,21 @@
 
 /** A class that handles converting entities to json format. */
 public class JsonFormatter {
-  public static String getCommentsJson(List<Comment> comments) {
+  public static String getCommentsJson(List<Comment> comments, List<User> users) {
     Gson gson = new Gson();
     List<Map<String, Object>> commentMapList = new ArrayList<Map<String, Object>>();
```

Comment:
maybe just like this is better?
```suggestion
    List<Map<String, Object>> commentMapList = new ArrayList<>();
```